boards/openmote-b/doc.txt

```diff
@@ -40,6 +40,11 @@ RIOT support flashing with USB by default.
 You may have to specify the flashing port with
 `PORT_DEV=<my_openmote_port> make flash`
 
+The flash tool needs to convert the generated Hex file to RAW format.
+For this it needs to have the intelhex library installed.
+
+    apt install python3-intelhex
```

Comment:
It seems like the `python3-intelhex` is not available for my `apt install` on ubuntu 16.04.

Maybe add "or `pip3 install intelhex`?

Reply:
Ok